modules/sd_hijack.py

```diff
@@ -356,8 +367,7 @@ def process_tokens(self, remade_batch_tokens, batch_multipliers):
             z = outputs.last_hidden_state
 
         # restoring original mean is likely not correct, but it seems to work well to prevent artifacts that happen otherwise
-        batch_multipliers_of_same_length = [x + [1.0] * (75 - len(x)) for x in batch_multipliers]
-        batch_multipliers = torch.asarray(batch_multipliers_of_same_length).to(device)
+        batch_multipliers = torch.asarray(batch_multipliers).to(device)

```

Comment:
why

Reply:
len(x) will always be <= 0, so it doesn't do anything. Line 207 pads tokens + multipliers to mults of 75.

Reply:
try a prompt with [a:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb:0.5] with more stuff before it so that first one is 75 characters and the second is 150

Reply:
like: `, , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , [, :, , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , :0.5]`? (in counts of `74 [1:76]`)?
It generates and works fine.

Using actual prompts + the OR feature: (view details)

<details>

```
masterpiece, best quality, cute girl, frilly dress, [white hair OR:black hair OR, OR:***a***]
Negative prompt: lowres, bad anatomy, bad hands, text, error, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality, normal quality, jpeg artifacts,signature, watermark, username, blurry, artist name
Steps: 15, Sampler: Euler, CFG scale: 11, Seed: 2008226614, Size: 512x640, Model hash: 925997e9, Eta: 0.69, Clip skip: 2
```

a = 0.1:

![0_1](https://user-images.githubusercontent.com/112723046/196231415-3a58390b-c88a-402f-8df6-2f2044c2f835.png)

a = 0.5:
![0_5](https://user-images.githubusercontent.com/112723046/196231427-8ba932d3-00e5-4999-8fe6-fe29b4b87c0d.png)

</details>

So it works as expected with the change.